Majasdarbs/10-Galadarbs/public/script.js

```diff
@@ -0,0 +1,48 @@
+//const userInput = document.getElementById("cityinput").value;
+//const userInput = document.querySelector("#cityinput");
+
+const userInput = "Riga";
+const KELVIN = 273;
+//const key = process.env.API_KEY;
+const key = "3045dd712ffe6e702e3245525ac7fa38";
+
+function getInputFromTextBox() {
+  let input = document.getElementById("cityinput").value;
+  console.log(input);
+}
+//Get weather from API provider
+async function start() {
+  try {
+    const response = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${userInput}&appid=${key}`
+    );
+    const data = await response.json();
+    weather.iconId = data.weather[0].icon;
+    console.log(data);
+    renderData(data);
+  } catch (e) {
+    console.log("There was a problem fetching data.");
```

Comment:
you can alert this (or show it inside the dom), for user to see the error message